proto/answer.proto

```diff
@@ -78,17 +52,29 @@ message ReadableConceptTree {
             repeated Node list = 1;
         }
 
-        message ReadableConcept {
-            oneof readable_concept {
-                EntityType entity_type = 1;
-                RelationType relation_type = 2;
-                AttributeType attribute_type = 3;
+        message Leaf {
+            oneof leaf {
+                Empty empty = 1;
 
-                RoleType role_type = 4;
+                EntityType entity_type = 10;
+                RelationType relation_type = 11;
+                AttributeType attribute_type = 12;
+                RoleType role_type = 13;
+                ValueType value_type = 15;
```

Comment:
We will at some point (soon) introduce `fetch`-specific shorthands to retrieve more than the 'basic' information, such as `type`, `value_type`, and `kind`:
```
fetch {
  "x-type": type($x),
  "x-kind": kind($x),
  "attr-value-type": value_type($attr),
  "type kind": kind($type_variable)
  }
 ```
Encoding these as a general `Leaf` is consistent. Now our `ConceptDocument.Node` enum is fully "structural", ie only contains Lists, Maps, or Leaves, where everything that can be a leaf is some kinds of concepts, descriptions (like 'kind' or 'value_type'), or Empty.